frontend/src/components/Login.js

```diff
@@ -0,0 +1,104 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import { Button } from './Button'
+import { Link, useHistory } from 'react-router-dom'
+
+const Section = styled.section`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+    a {
+      text-decoration: none;
+    }
+`
+
+const Form = styled.form`
+  display: flex;
+  flex-direction: column;
+    label {
+      margin: 10px;
+      color:  #254b62;
+    }
+    input {
+      width: 100%;
+      height: 20px;
+      border-radius: 6px;
+    }
+`
+
+const Header = styled.h1`
+  color:  #254b62;
+  text-shadow: 2px 2px #cff532;
+  font-size: 40px;
+  margin-top: 40px;
+`
+
+const Text = styled.h2`
+  color:  #254b62;
+  font-size: 20px;
+  margin-top: 40px;
+  margin-bottom: 1px;
+`
+
+export const Login = () => {
+  const [signInValues, setSignInValues] = useState({
+    email: '',
+    password: ''
+  })
+  const history = useHistory()
+  const [error, setError] = useState('')
+  const url = 'https://anna-project-auth.herokuapp.com/sessions'
+
+  // Posts email and password to the api
+  const handleLogin = event => {
+    event.preventDefault()
+    fetch(url, {
+      method: 'POST',
+      body: JSON.stringify(signInValues),
+      headers: { 'Content-Type': 'application/json' }
+    })
+      .then(res => {
+        if (!res.ok) {
+          throw new Error('Could not find your user.')
+        }
+        res.json().then(data => {
+          if (data.notFound !== true) {
```

Comment:
This becomes a little hard to follow, with the double negatives. Is it possible to write this as a positive statement instead?